internal/page/fixtures/attorney.go

```diff
@@ -296,6 +304,22 @@ func Attorney(
 		if progress >= slices.Index(progressValues, "signedByCertificateProvider") {
 			donorDetails.SignedAt = time.Now()
 			certificateProvider.SignedAt = donorDetails.SignedAt.Add(time.Hour)
+
+			if withTestSchedule {
+				if err = scheduledStore.Create(attorneyCtx, scheduled.Event{
+					At:                time.Now(),
+					Action:            scheduleddata.ActionRemindAttorneyToComplete,
+					TargetLpaKey:      donorDetails.PK,
+					TargetLpaOwnerKey: donorDetails.SK,
+					LpaUID:            donorDetails.LpaUID,
+				}); err != nil {
+					return fmt.Errorf("error scheduling attorneys prompt: %w", err)
+				}
+
+				donorDetails.AttorneysInvitedAt = time.Now().AddDate(0, -4, 0)
+				donorDetails.IdentityUserData.CheckedAt = time.Now()
+				donorDetails.SignedAt = time.Now().AddDate(-1, -10, 0)
```

Comment:
My brain just doesn't compute the logic in the scheduler very well with dates to check so there may be a more appropriate date that satisfies it than this!